cpp/CPP14Lexer.g4

```diff
@@ -347,6 +352,7 @@ fragment Simpleescapesequence:
 	| '\\f'
 	| '\\n'
 	| '\\r'
+    | ('\\' ('\r' '\n'? | '\n'))
```

Comment:
Align spaces with other alternatives.

Reply:
I think that's a tab rather than four spaces:
```suggestion
	| ('\\' ('\r' '\n'? | '\n'))
```